### PR TITLE
Sfr 81 sort by title

### PR DESCRIPTION
**Pull request by nachocs**
Added tests for all links and selects of searchnavigation
sortBy Map and list of numbersByPage in separate file.
Added data-test for easier testing

### CODE REVIEW

**Comment by nachocs** [comment]
I saw data-test as something commonly used, only for helping in testing. Chris suggested to add classes, but I think this approach is more solid for this purpose. Anyway here it's just a check to see what everybody think about this.

File: src/app/constants/sorts.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import allowedFilters from './filters';
-
-const sorts = {
-  filter: allowedFilters,
-  order: ['asc', 'desc'],
+export const sortMap = {
+  Relevance: [],
+  'Title A-Z': [{ field: 'title', dir: 'ASC' }],
```
Comment (seanredmond) [inline src/app/constants/sorts.js]: Dash is missing in render (checked Firefox and Chrome)

Screen Shot 2019-06-07 at 10 01 03 AM

Comment (nachocs) [reply src/app/constants/sorts.js]: Solved

Comment (seanredmond) [reply src/app/constants/sorts.js]: Confirmed that its working for me

**Comment by seanredmond** [comment]
Just to record a conversation: I agree in general with @gkallenberg about mixing testing variables and application code. In this case it isn't necessary because the element the `data-test` attribute has been added to has an `id` that @chrismulholland can use to target tests.

**Comment by nachocs** [comment]
Removed data-test and updated

**Pull request merged**